File: ESC.Resturant.Data/Entities/BoxMonyType.cs
using ESA.Data;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ESC.Resturant.Data.Entities
{
    public class BoxMonyType
    {

        [Key]
        public int Id { get; set; }
        public string NameAR { get; set; }
        public string NameEN { get; set; }
        public virtual ICollection<BoxMonies> BoxMonies { get; set; }
        public virtual ICollection<ApplicationUser> User { get; set; }
    }
}

File: ESAResturant.NEW.web/ClientApp/src/app/admin/BoxMony/Models/boxMonyType.ts
import { boxMony } from '../Models/boxMony';
import { User } from '../../../users/Models/User';


export class boxMonyType {

  constructor() {

    this.id = 0;
    this.nameAR = "";
    this.nameEN = "";
    this.user = [];
  }


  id: number;
  nameAR: string;
  nameEN: string;
  boxMony: boxMony;
  user: User[];
}
